src/pages/Home/index.tsx

```diff
@@ -1,15 +1,26 @@
 import { Container } from 'react-bootstrap';
 import { useEffect, useState } from 'react';
 import { OrderProps, BobaProps } from 'types/common/main';
-import { bobaList, toppingsList, milkList } from 'assets/sampleBobaAPI';
 import BobaContainer from 'components/BobaContainer';
 import SearchBar from 'components/SearchBar';
 import CustomizeModal from 'components/Modal';
+import Loading from 'components/Loading';
 import BobaModalForm from './CustomizeBobaModalBody';
 import Button from 'react-bootstrap/Button';
+import useApi from 'hooks/API';
 
 const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
-  const [filteredBobaList, setFilteredBobaList] = useState<BobaProps[]>(bobaList);
+  const { data, error, loading } = useApi(
+    `${process.env.REACT_APP_BOBA_FETCH}/boba_list` as string,
+  );
+
+  const { data: toppingsList } = useApi(
+    `${process.env.REACT_APP_BOBA_FETCH}/toppings_option` as string,
```

Comment:
Update `useApi` with 2 params for host and endpoint

Reply:
updated at 548ae7f07d07dcbfbf4d21fc11805b1ab3a7594a